deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -226,8 +346,9 @@ spec:
   keywords:
   - windows
   maintainers:
-  - {}
-  maturity: alpha
+    - email: team-winc@redhat.com
+      name: Red Hat Windows Containers Team
+  maturity: preview
```

Comment:
This is misusing the maturity field I believe https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#operator-metadata

Reply:
The channel is defined in the dockerfile not within the CSV

Reply:
I suggest using `stable` given we are releasing in a week.